.mise.toml

```diff
@@ -22,5 +22,6 @@ bun = "latest"
 python = '3.11'
 uv = "0.6.16"
 "go:github.com/mitranim/gow" = "latest"
+"go:go.abhg.dev/requiredfield/cmd/requiredfield" = "latest"
```

Comment:
This should match the version we have in CI. Either we make both `latest`, or both `v0.7.1`.

Reply:
Ah yeah good catch.